HPXMLtoOpenStudio/measure.rb

```diff
@@ -4094,10 +4093,11 @@ def self.get_space_temperature_schedule(model, location, spaces)
 
     sch = OpenStudio::Model::ScheduleConstant.new(model)
     sch.setName(location)
+    heating_setpoint = @hpxml.hvac_controls[0].heating_setpoint_temp
```

Comment:
This simply grabs base setpoint temperature without considering setback. @shorowit Good enough?

Reply:
Is it easy to use a sensor on the heating setpoint schedule value? If not, then this seems fine as an approximation.

Reply:
Unfortunately the sequence of calling add_setpoints method is after surface creation right before adding hvac, so the heating setpoint schedule is not there at this point. Simply grabbing heating setpoint from OS model is not working. But we can manually pass the `Constants.ObjectNameHeatingSetpoint` as sensor name, which is obviously not that safe.

Reply:
I can't think of any reason why we couldn't move the add_setpoints call to before the surface creation. Is that worth trying?

Reply:
Sure, I can try changing its calling sequence.